rspace/src/main/scala/coop/rchain/rspace/LockFreeInMemoryStore.scala

```diff
@@ -100,7 +100,7 @@ class LockFreeInMemoryStore[F[_], T, C, P, A, K](
     entriesGauge.set(stateGNAT.readOnlySnapshot.size.toLong)
 
   private[rspace] def hashChannels(channels: Seq[C]): Blake2b256Hash =
-    StableHashProvider.hash(channels)
+    StableHashProvider.hash(channels)(Serialize.fromCodec(codecC))
```

Comment:
wow. What other Serialize instance kicks in? Maybe the other instance should be removed as well?